pages/common/sort.md

```diff
@@ -10,6 +10,10 @@
 
 `sort -r {{filename}}`
 
+- Sort a file in case-insensitive way:
+
+`sort -f {{filename}}`
```

Comment:
I would recommend to use the longer version - `--ignore-case` because it is easier to understand. `-f` stands for fold which means we would have to use that word in the description.